x-pack/plugin/inference/build.gradle

```diff
@@ -57,9 +57,11 @@ dependencies {
   implementation 'io.grpc:grpc-context:1.49.2'
   implementation 'io.opencensus:opencensus-api:0.31.1'
   implementation 'io.opencensus:opencensus-contrib-http-util:0.31.1'
+  implementation "org.apache.commons:commons-lang3:${versions.commons_lang3}"
```

Comment:
Both of these are needed for the Apache commons `StringSubstitutor` being used for replacing the placeholder.